packages/store/addon/-private/system/core-store.ts

```diff
@@ -13,7 +13,7 @@ import { default as RSVP, all, resolve, Promise, defer } from 'rsvp';
 import Service from '@ember/service';
 import { typeOf, isPresent, isNone } from '@ember/utils';
 
-import require, { has } from 'require';
+import require from 'require';
```

Comment:
Just curious, what are we using `require` for here now?

Reply:
if we do have the model package we have to import the model class for `instanceof` checks